graphql/data/schema.js

```diff
@@ -2066,6 +2067,22 @@ const deleteUserMutation = mutationWithClientMutationId({
   },
 })
 
+const updateMissionNotificationMutation = mutationWithClientMutationId({
+  name: 'UpdateMissionNotification',
+  inputFields: {
+    userId: { type: new GraphQLNonNull(GraphQLString) },
+    missionId: { type: new GraphQLNonNull(GraphQLString) },
+  },
```

Comment:
could add something like updatefield: { type graphqlEnum} with acknowledgeMissionStarted and acknowledgeMissionCompleted

Reply:
sounds good.